runtime/compiler/codegen/J9CodeGenerator.cpp

```diff
@@ -798,7 +798,8 @@ J9::CodeGenerator::lowerTreeIfNeeded(
             storeNode = TR::Node::createStore(vmThreadTempSlotSymRef, numArgsNode, TR::istore);
             }
          storeNode->setByteCodeIndex(node->getByteCodeIndex());
-         TR::TreeTop::create(self()->comp(), tt->getPrevTreeTop(), storeNode);
+         TR::TreeTop * storeTT = TR::TreeTop::create(self()->comp(), storeNode);
+         tt->getEnclosingBlock()->prepend(storeTT);
```

Comment:
this might need re-visiting after #12848 is merged.

Reply:
Can you elaborate?  12848 is merged.

Reply:
The reason this change was originally made was because `TR::TreeTop::create(self()->comp(), tt->getPrevTreeTop(), storeNode);` inserted the `storeTT` in the computed static call block instead of the block containing the INL call, resulting in that store not being performed before the j2i transition. `tt->getEnclosingBlock()->prepend(storeTT)` was a workaround to ensure this gets inserted in the right place. Perhaps this unexpected behaviour was caused because the INL call node was not anchored using a TreeTop node initially during my tests. I wanted to revisit this after 12848 was merged to ensure the transformations introduced here does not cause the same issue with the store to `floatTemp1` and if we could get rid of the change in this file given that the missing TreeTop has been fixed now.